src/schema/v2/fairs.ts

```diff
@@ -83,3 +87,72 @@ const Fairs: GraphQLFieldConfig<void, ResolverContext> = {
 }
 
 export default Fairs
+
+export const fairsConnection = {
+  type: fairConnection.connectionType,
+  args: pageable({
+    fairOrganizerID: { type: GraphQLString },
```

Comment:
Are all of these params needed?

Reply:
Probably not but it's not a terrible idea just to support whatever the other fairs field supports.